extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -61,6 +64,10 @@ class AmpStickyAd extends AMP.BaseElement {
         'amp-sticky-ad must have a single amp-ad child');
 
     this.ad_ = children[0];
+    const adType = this.element.getAttribute('type');
+    const config = adConfig[this.ad_.getAttribute('type')];
```

Comment:
I'd much rather add a method on the `<amp-ad>` to tell whether it uses `renderStart`. This kind of code does not belong in `<amp-sticky-ad>`.